pontoon/base/static/js/translate.js

```diff
@@ -35,18 +35,31 @@ var Pontoon = (function (my) {
         markPlaceables
       );
 
-      var translationString = self.fluent.getSimplePreview(
-        translation.string,
-        self.markPlaceables(translation.string || ''),
-        markPlaceables
-      );
+      var translationString = '';
+      if (!translation.rejected) {
+        translationString = self.fluent.getSimplePreview(
+          translation.string,
+          self.markPlaceables(translation.string || ''),
+          markPlaceables
+        );
+      }
 
-      var li = $('<li class="entity' +
-        (' ' + status) +
-        (translation.pk ? ' has-translations' : '') +
-        (!entity.body ? ' uneditable' : '') +
-        (this.allEntitiesSelected ? ' selected' : '') +
-        (entity.visible ? ' visible': '') +
+      var classNames = ['entity', status];
+      if (translation.pk && !translation.rejected) {
```

Comment:
I'm not sure about this change. It looks like this: 

![](https://screenshotscdn.firefoxusercontent.com/images/631e61fb-5531-4e2b-8c0d-65f03d95860a.png)

I think it's ugly, I'd rather always have strings take only half of the space so that it's quite clear which ones lack a translation.